cpp/modmesh/view/RParameter.cpp

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (c) 2023, Buganini Chiu <buganini@b612.tw>
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * - Redistributions of source code must retain the above copyright notice,
+ *   this list of conditions and the following disclaimer.
+ * - Redistributions in binary form must reproduce the above copyright notice,
+ *   this list of conditions and the following disclaimer in the documentation
+ *   and/or other materials provided with the distribution.
+ * - Neither the name of the copyright holder nor the names of its contributors
+ *   may be used to endorse or promote products derived from this software
+ *   without specific prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+ * ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+ * LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+ * CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+ * SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+ * INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+ * CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+ */
+
+#include <modmesh/view/RParameter.hpp> // Must be the first include.
+
+namespace modmesh
+{
+
+static int int64V = 5566;
+static double doubleV = 77.88;
+
+static std::shared_ptr<pybind11::object> paramsState = nullptr;
+static std::shared_ptr<pybind11::list> paramsList = nullptr;
+
+template <typename T>
+void addParam(const char *key, T *ptr) {
```

Comment:
Would it be possible to store the parameter information in the hierarchical dynamic toggle: https://github.com/solvcon/modmesh/blob/1bd05322720e3ce4064c7c36acccd44081c68bf1/tests/test_toggle.py#L280 ?

More information about modmesh toggles: https://github.com/solvcon/modmesh/blob/1bd05322720e3ce4064c7c36acccd44081c68bf1/cpp/modmesh/toggle/toggle.hpp#L220